examples/wearable/UIComponents/contents/navigationelements/pageindicator/pageindicator-circular.js

```diff
@@ -19,6 +19,7 @@
 		pageIndicator.setActive(0);
 		// make SectionChanger object
 		sectionChanger = new tau.widget.SectionChanger(changer, {
+      circular: true,
```

Comment:
wrong indentation

Reply:
This property indicates that changing of section is in loop... I have on mind the after last item section changer goes to first section. Is this intended change?

Reply:
Yes, it is.
There was no difference between the two existing examples(pageindicator-horizontal & pageindicator-circle).
So I change from the pageindicator-circle to the pageindicator-circular.
Do you know the difference between the two previous examples?

Reply:
Difference is in behaviour. For `circular=false`, when you change section to the last section you cannot change more. You can change only back. Otherwise, it loops to the first.

Reply:
That's what I intended.
Anyway, I modified it to fit the content.